packages/domain/lib/use_case/login_validator_use_case.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:domain/model/firebase_user_email.dart';
+import 'package:domain/model/validate_model.dart';
+import 'package:domain/use_case/sample_use_case/use_case_in_out.dart';
+
+const _patternRegexLogin = '^.{8,}\$';
+const _patternRegexPassword = '^[^A-Z][^a-z]{6,}\$';
```

Comment:
seems that regex is wrong. it accepts only symbols, which are not letters, no?

Reply:
in my head translated as symbols and numbers)